src/components/Modals/FilterModal/FilterModal.tsx

```diff
@@ -42,21 +51,27 @@ const FilterModal = ({ isModalOpen, onRequestClose }: TFilterModalProps) => {
         {
             component: <LabelPairedChevronRightLgRegularIcon />,
             onClick: () => setShowPaymentMethods(true),
-            subtext: paymentMethodNames,
+            subtext: paymentMethodNames || localize('All'),
```

Comment:
For some reason, the All text wasn't being shown here:

![Screenshot 2024-10-08 at 4 03 02 PM](https://github.com/user-attachments/assets/79d44416-f027-402d-8173-3964d3b3f85b)
![Screenshot 2024-10-08 at 4 03 11 PM](https://github.com/user-attachments/assets/43342e79-5c21-4191-87ae-db1a7eee7dbb)

Reply:
it think it should show as 'all' only when all the payment methods are checked..

Reply:
in the Figma Design it was 'All' 😓 
<img width="528" alt="Screenshot 2024-10-11 at 3 26 07 PM" src="https://github.com/user-attachments/assets/243412a1-b9d5-4e99-a68b-51433dcdba75">

Reply:
yeah.. may be its assumed that all the payment methods were selected.. in v1 as well, we show the 'all' only when we select all the payment methods.. because if we select all , then the ones without PMs wont be listed..

Reply:
i think in this case, they did this as "All" it just means everything by default, but it is odd to have this 👀